db/migrate/20191029210303_create_auctions.rb

```diff
@@ -0,0 +1,14 @@
+class CreateAuctions < ActiveRecord::Migration[6.0]
+  def change
+    create_table :auctions do |t|
+      t.string :title
+      t.string :description
+      t.decimal :start_bid, precision: 10, scale: 2
```

Comment:
This is similar to the comment left by @ijpiantanida, but I would avoid adding columns you are not using yet.

One bug issue is that you are making assumptions about how the data is going to be used and stored. For example, I might expect bids to be stored separately than auctions, and getting the current highest bid to be more complex than just looking up a column on the auctions table.